tests/integration/psql_import/test_import_psql.py

```diff
@@ -287,3 +287,89 @@ def test_start_up(docker_compose):
             return_results=True,
         )
         assert len(results[0].docs[0].matches) == len(docs)
+
+
+@pytest.mark.parametrize('shards', [1])
+@pytest.mark.parametrize('nr_docs', [100])
+@pytest.mark.parametrize('emb_size', [10])
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
+def test_psql_sync_delta(
+    tmpdir,
+    nr_docs,
+    emb_size,
+    shards,
+    docker_compose,
+):
+    # for psql to start
+    time.sleep(2)
+
+    top_k = 5
+    batch_size = min(1000, nr_docs)
+    docs_original = get_batch_iterator(
+        batches=nr_docs // batch_size, batch_size=batch_size, emb_size=emb_size
+    )
+    docs_original = DocumentArray(flatten(docs_original))
+    assert len(docs_original) == nr_docs
+
+    os.environ['STORAGE_WORKSPACE'] = os.path.join(str(tmpdir), 'index_ws')
+    os.environ['SHARDS'] = str(shards)
+    uses_after = 'Pass'
+
+    with _flow(uses_after=uses_after, total_shards=shards, startup_args={}) as flow:
+        flow.post(on='/index', inputs=docs_original)
+
+        results = flow.post(
+            on='/search',
+            inputs=get_documents(nr=1, index_start=0, emb_size=emb_size),
+            return_results=True,
+        )
+        assert len(results[0].docs[0].matches) == 0
+
+        flow.post(
+            on='/snapshot',
+        )
+
+        flow.post(
+            on='/sync',
+        )
+
+        results = flow.post(
+            on='/search',
+            inputs=get_documents(nr=3, index_start=0, emb_size=emb_size),
+            parameters={'top_k': top_k},
+            return_results=True,
+        )
+        assert len(results[0].docs[0].matches) == top_k
+        assert results[0].docs[0].matches[0].scores[METRIC].value > 0.0
```

Comment:
Seems to me we are testing `search` here - this should be done (maybe already is) in a different test

Reply:
You are testing the case `'use_delta': True` in this test. Testing normal `sync` without it should be done as a separate test